<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>441</width>
      <height>489</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Gerar valores de 1 até:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_imprimir_iterativo">
             <property name="text">
              <string>I</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="QLineEdit" name="lineEdit_imprimir"/>
           </item>
           <item>
            <widget class="QPushButton" name="btn_imprimir_recursivo">
             <property name="text">
              <string>R</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Gerar valores pares de 1 até:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_par_iterativo">
             <property name="text">
              <string>I</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QLineEdit" name="lineEdit_par"/>
           </item>
           <item>
            <widget class="QPushButton" name="btn_par_recursivo">
             <property name="text">
              <string>R</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Gerar valores ímpares de 1 até:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_impar_iterativo">
             <property name="text">
              <string>I</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLineEdit" name="lineEdit_impar"/>
           </item>
           <item>
            <widget class="QPushButton" name="btn_impar_recursivo">
             <property name="text">
              <string>R</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Somar valores de 1 até</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_somar_iterativo">
             <property name="text">
              <string>I</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLineEdit" name="lineEdit_somar"/>
           </item>
           <item>
            <widget class="QPushButton" name="btn_somar_recursivo">
             <property name="text">
              <string>R</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Gerar valor de pi 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="btn_pi_iterativo_1">
             <property name="text">
              <string>I</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_pi_recursivo_1">
             <property name="text">
              <string>R</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Gerar valor de pi 2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="btn_pi_iterativo_2">
             <property name="text">
              <string>I</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_pi_recursivo_2">
             <property name="text">
              <string>R</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextEdit" name="textEdit_imprimir"/>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_par"/>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_impar"/>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_somar"/>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_pi_1"/>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_pi_2"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
